packages/universal-components/src/Button/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -32,22 +32,31 @@ exports[`Button - native should match snapshot diff 1`] = `
 +             \\"justifyContent\\": \\"space-between\\",
             },
             false,
+            Object {
+              \\"height\\": 44,
+-             \\"paddingEnd\\": 16,
+-             \\"paddingStart\\": 16,
++             \\"paddingEnd\\": 12,
++             \\"paddingStart\\": 12,
```

Comment:
This is ok and you can remove your note from description. It's difference snapshot. So padding is not added twice but it is simply just difference between button with and without icons.

Reply:
Thank you for explanation! It was so confusing